test/runtests.jl

```diff
@@ -12,51 +13,27 @@ Pkg.develop(PackageSpec(path=joinpath(filter_path, "H5Zzstd")))
 # To debug HDF5.jl tests, uncomment the next line
 # ENV["JULIA_DEBUG"] = "Main"
 
-@testset "HDF5.jl" begin
-
-@debug "plain"
 include("plain.jl")
```

Comment:
The same can be achieved with the removal of the `@testset "HDF5.jl"` block, so that specific `Test Summary: | Pass  Broken  Total` appear when using `include`. I believe that the top level `@testset` doesn't add anything and has the drawback of hiding the test sequence. With the removal you also track down hangs: https://github.com/JuliaIO/HDF5.jl/runs/7194199530?check_suite_focus=true#step:5:159.

Reply:
Yes, I understand. However, we only really wanted it enabled during debugging and for CI. The multiprocess setup of `Pkg.test` also creates some unusual interference. Let's discuss in another pull request, please.